src/modules/App/App.js

```diff
@@ -76,6 +75,14 @@ const App = () => {
                 return <MissionControl id={id}/>;
                }}
             />
+            <Route
```

Comment:
Glad we were able to implement this! Let's make sure the server is working for the redeem rewards.